### PR TITLE
Update lecture 7 (try 2)

### PR DESCRIPTION
**Pull request by sidd3888**
- Have removed a block of code that runs the `lax.scan` implementation before it is defined.
- Fixed a typo that wrongly runs the `for` loop implementation after JIT compilation of the `lax.scan` version.
- Stated the initial parameters to be used (inferred from solution) for exercise 7.2.

All changes identical to the previous PR but made on a local branch instead

### CODE REVIEW

**Comment by mmcky** [comment]
thanks for transferring the edits to a local branch @sidd3888 

@jstac this replaces #90

**Comment by jstac** [comment]
Many thanks @sidd3888 .

@HumphreyYang , could you please do a preliminary review of this PR and then pass it back to me?

File: lectures/wealth_dynamics.md [hunk 1]
```diff
@@ -592,6 +582,11 @@ this what you see?
 
 For sample size and initial conditions, use
 
+```{code-cell} ipython3
+num_households = 250_000
+T = 500 #shift forward T periods
```
Comment (HumphreyYang) [inline lectures/wealth_dynamics.md:587]:
```suggestion
T = 500 # Shift forward T periods
```

File: lectures/wealth_dynamics.md [hunk 2]
```diff
@@ -592,6 +582,11 @@ this what you see?
...
+ψ_0 = jnp.full((num_households, ), wdy.y_mean) # initial distribution
```
Comment (HumphreyYang) [inline lectures/wealth_dynamics.md:588]:
```suggestion
ψ_0 = jnp.full((num_households, ), wdy.y_mean) # Initial distribution
```

**Comment by HumphreyYang** [comment]
Hi @jstac,

These changes look great to me. Please feel free to commit these small suggestions directly and merge them.  I think we will also need a new publish after merging (CC @mmcky). 

The error is introduced in #62. I think we might need to have two sets of eyes before merging a PR : )

Many thanks @sidd3888 again for pointing out this typo!

**Comment by jstac** [comment]
Thanks @HumphreyYang for the review and suggestions.

@sidd3888 , could you please make these small changes.

My bad for merging the faulty PR.  @mmcky I think @HumphreyYang is right that we should try to get two sets of eyes over each PR.  Maybe we can systematize this at some point in the future...

**Comment by mmcky** [comment]
@HumphreyYang 

> The error is introduced in https://github.com/QuantEcon/lecture-jax/pull/62. I think we might need to have two sets of eyes before merging a PR : )

Is this error causing execution failure? Or is this an error in logic? Just thinking through the infrastructure side if it failed to warn us.

**Comment by HumphreyYang** [comment]
> Is this error causing execution failure? Or is this an error in logic? Just thinking through the infrastructure side if it failed to warn us.

It was a NameError:

```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
File <timed exec>:2

NameError: name 'wealth_time_series_scan_jax' is not defined
```

I am also curious why it did not fail CI.

**Comment by sidd3888** [comment]
@jstac I have committed the changes suggested by @HumphreyYang. Should be good to go now?

**Comment by jstac** [comment]
Many thanks @sidd3888  and thanks also @HumphreyYang for the review.  I'll merge this now.

@mmcky I'll go ahead and merge but just noting that we seem to have had an execution failure that didn't get picked up.  (Unless I was very tired and merged a failing PR, but I can't imagine doing that...)

**Comment by mmcky** [comment]
@jstac I think it has something to do with the cell magic and not returning the `NamedError` the same way. Still trying to solve that mystery. The PR passed just fine BTW (that you merged) so the `jb` didn't fail.

**Pull request merged**